packages/client/tests/functional/issues/8832/_matrix.ts

```diff
@@ -0,0 +1,9 @@
+import { defineMatrix } from '../../_utils/defineMatrix'
+
+export default defineMatrix(() => [
+  [
+    {
+      provider: 'postgresql',
```

Comment:
Why do we only test for PostgreSQL here?

This comment https://github.com/prisma/prisma/issues/8832#issuecomment-1237850099 mentions that the limit is different on other databases, maybe it would be good to test these as well?

Reply:
Regression was only identified for PostgreSQL. We created an issue to test others as well (it is mentioned somewhere else here)

Reply:
Seems issue was only created, not mentioned here at all: https://github.com/prisma/prisma-private/issues/192